docs/source/schema-design/federated-schemas/entities/intro.mdx

```diff
@@ -1,24 +1,24 @@
 ---
 title: Introduction to Entities
-subtitle: Resolve federated types across multiple subgraphs
-description: Learn to define, contribute to, and reference object types that resolve their fields across multiple subgraphs in a federated GraphQL architecture.
+subtitle: Fundamental keyed object type of Apollo Federation
```

Comment:
Not sure why, but I'm not seeing the new subtitle in the preview: https://www.apollographql.com/docs/deploy-preview/2601ef076b8655656511bee3/graphos/schema-design/federated-schemas/entities/intro#defining-an-entity

Reply:
weird, same for me, and the subtitle was updated in the first commit of this branch, so wouldn't think it's a preview redeployment issue, gonna check local deploy ...

Reply:
Hopefully this redeploy does the trick.